docs/extensions.md

```diff
@@ -148,6 +148,40 @@ To get real-time data for providers who aren't using Open Data, follow the instr
 
 ---
 
+## Unlocking geocoding providers
+
+To get results from geocoding data for providers who aren't using Open Data when using `geocoding.json` endpoint, follow the instructions for the provider below. If you want to use a provider that hasn't yet been connected to our system, please get in touch with our team [by mail](mailto:api@skedgo.com) or on [Slack](http://slack.tripgo.com/) (by [self-invite](http://slackin.tripgo.com/)).
```

Comment:
Please clarify if this also applies to autocompletion (same endpoint, but an additional parameter), or just geocoding.